.github/workflows/ci.yml

```
@@ -33,35 +29,53 @@ jobs:
       matrix:
         version: [20, 22]
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
       - name: Using Node.js ${{ matrix.version }}
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@v5
         with:
           node-version: ${{ matrix.version }}
       - name: Install dependencies
         run: npm install
       - name: Run tests
         run: npm test
 
-  build_and_publish:
-    needs: [test]
+  build:
     runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - uses: actions/checkout@v5
+
+      - uses: actions/setup-node@v5
+        with:
+          node-version: 22
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run build
+
```

We should make sure that tests pass before publish, shouldn't we?

```suggestion

      - name: Test
        run: npm run test

```

---

No worries 😄. I'm curious to know if it would work. I never did this before. Fingers crossed🤞